utils/src/main/java/org/apache/cloudstack/utils/redfish/RedfishClient.java

```diff
@@ -265,12 +265,12 @@ private String getRequestPathForCommand(RedfishCmdType cmd, String resourceId) {
             if (StringUtils.isBlank(resourceId)) {
                 throw new RedfishException(String.format("Command '%s' requires a valid resource ID '%s'.", cmd, resourceId));
             }
-            return String.format("%s%s", SYSTEMS_URL_PATH, resourceId);
```

Comment:
cc @wido @GabrielBrascher Line #263 is where I worry if not having a trailing slash `/` may break any redfish servers/bmcs ?

Reply:
so line 263 could read
```
return SYSTEMS_URL_PATH + "/";
```
o.r use File.SEPARATOR and/or String.format(). I think we should test it to make sure, but would not be a big issue

Reply:
No that's not what I meant - if for the basic discovery of the endpoint, a trailing slash isn't passed some systems could fail (hypothetically, I'm just asking). I couldn't return with the `/` as it breaks the base case per Kiran's findings (if I understood them right)

Reply:
ok, so do you mean we may have to spilt this case out into different ones?

Reply:
TBH, I don't know. Usually, I don't think we should require a trailing `/` (most servers should handle this case).